apps/cowswap-frontend/src/modules/appData/utils/buildAppData.ts

```diff
@@ -95,24 +97,13 @@ export async function updateHooksOnAppData(
 ): Promise<AppDataInfo> {
   const { doc } = appData
 
-  const existingHooks = filterHooks(doc.metadata.hooks, preHooksFilter, postHooksFilter)
-
-  const mergedHooks = mergeHooks(existingHooks, hooks)
-  const noDuplicateHooks = mergedHooks
-    ? {
-        ...mergedHooks,
-        ...(mergedHooks.pre ? { pre: removeDuplicatedHook(mergedHooks.pre) } : undefined),
-        ...(mergedHooks.post ? { post: removeDuplicatedHook(mergedHooks.post) } : undefined),
-      }
-    : mergedHooks
-
-  console.log(`bug:updateHooksOnAppData`, noDuplicateHooks)
+  const filteredHooks = filterHooks(doc.metadata.hooks, preHooksFilter, postHooksFilter)
```

Comment:
We shouldn't merge it here, because at this point we can't tell which hook is what

Reply:
Although this might have broken the hook store logic. Will need to address that, but probably in a different PR.

Reply:
Fixed the merging logic